package.json

```diff
@@ -21,7 +25,7 @@
   },
   "devDependencies": {
     "@types/node": "^16.3.3",
-    "@types/react": "^18.2.0",
+    "@types/react": "^18.0.28",
```

Comment:
Is there any reason for this?

Reply:
18.2.0 did not appear to exist, so it prompted me to choose a version manually every time I ran yarn..